rst_files/lake_model.rst

```diff
@@ -217,60 +217,76 @@ Setup
 
 .. code-block:: julia
 
-    function LakeModel(;λ = 0.283, α = 0.013, b = 0.0124, d = 0.00822)
+    LakeModel = @with_kw (λ = 0.283, α = 0.013, b = 0.0124, d = 0.00822)
+
+    function A_matrices(lm)
+        @unpack λ, α, b, d = lm
         g = b - d
-        A = [(1 - λ) * (1 - d) + b (1 - d) * α + b   # carry out intermediate calculations
-            (1 - d) * λ            (1 - d) * (1 - α)]
+        A = [(1 - λ) * (1 - d) + b      (1 - d) * α + b
+            (1 - d) * λ                 (1 - d) * (1 - α)]
         A_hat = A ./ (1 + g)
```

Comment:
Change `A_hat` to `Â` throughout.

This is a general comment.  For any annotations of ascii letters (e.g. bar, hat, etc.) use the appropriate glyph.  Sadly it doesn't work with annotations on greek letters/etc.

Reply:
Done.